src/status.rs

```diff
@@ -0,0 +1,121 @@
+use crate::queue::{Queue, SerdeSerializedSend};
+use chrono::Local;
+use serde::{Deserialize, Serialize};
+
+const DATE_FORMAT: &str = "%Y.%m.%d %H:%M:%S";
```

Comment:
```suggestion
const DATE_FORMAT: &str = "%Y-%m-%d %H:%M:%S";
```

sorry für die erneute Anpassung. So wäre es aber richtig nice